Vancouver-Elsevier-Spain

```diff
@@ -0,0 +1,156 @@
++<?xml version="1.0" encoding="utf-8"?>  
++<style class="in-text" demote-non-dropping-particle="sort-only" version="1.0" page-range-format="minimal" xmlns="http://purl.org/net/xbiblio/csl">  3 +  <info>  
++  <title>Vancouver-Elsevier-Spain</title>  
++    <id>http://www.zotero.org/styles/Vancouver-Elsevier-Spain</id>  
++    <link href="http://www.zotero.org/styles/Vancouver-Elsevier-Spain" rel="self"/>  
```

Comment:
But I want the style to have this name because other journals would not relate to Medicina-clinica. If I change these 3 things, will that change the name that appears?